java/code/src/com/redhat/rhn/taskomatic/task/MinionActionExecutor.java

```diff
@@ -44,8 +47,7 @@
  */
 public class MinionActionExecutor extends RhnJavaJob {
 
-    public static final int ACTION_DATABASE_GRACE_TIME = 600_000;
-    public static final int ACTION_DATABASE_POLL_TIME = 100;
+    public static final int MAX_RETRIES = 40;
```

Comment:
40 may be a high number, especially with the increase in the wait time. I think 20 should be enough (that should already give something like 2 minutes.

Reply:
I was making some calculations, and maybe we can have it on 25, wich will give more or less 5 minutes. That should be more then enough